src/app/account-app/scriptloader.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+
+// adopted from https://stackoverflow.com/a/42766146
+@Injectable()
+export class ScriptLoaderService {
+    scripts = {
+        stripe: { loaded: false, src: 'https://js.stripe.com/v3/' }
+    };
+
+    loadScript(name: string): Promise<void> {
+        return new Promise((resolve, reject) => {
```

Comment:
I think probably using an asyncsubject for script loading is better than promise and maintaining a tracking object. Like the scriptloader I've made for loading Xapian: https://github.com/runbox/runbox7/blob/master/src/app/xapian/xapianwebloader.ts#L37

Reply:
`AsyncSubject` may indeed make more sense. I created the tracking object because I thought that we might need this kind of dynamic-external-JS-loading in other places too. Now I see that we also needed it for xapian, and that my `scriptloader` is not quite good enough for it needs either :P So probably a YAGNI on my part, will remove it.

Reply:
Addressed in e88c0f8.